package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@types/sqlite3": "^3.1.6",
     "assert": "^2.0.0",
+    "body-parser": "^1.19.0",
```

Comment:
It would be better to separate the module into a different commit.

Reply:
I've separated it with two commit.